clang/test/3C/allarrays.c

```diff
@@ -8,7 +8,7 @@ void bar(void) {
   int *p = 0;
   //CHECK: _Array_ptr<int> p = 0;
   int *q = foo(p);
-  //CHECK: _Array_ptr<int> q = foo(p);
+  //CHECK: _Array_ptr<int> q : count(1 + 1) = foo(p);
```

Comment:
Do we know when emitting the +1 count bound that the underlying count is a constant bound? it seems to me that it shouldn't be hard to realize this and fold the constant expression. Not really an issue though.

Reply:
Yeah. Constant folding could be done, but not a priority now.